system-test/spanner.ts

```diff
@@ -45,6 +46,10 @@ const spanner = new Spanner({
   projectId: process.env.GCLOUD_PROJECT,
   apiEndpoint: process.env.API_ENDPOINT,
 });
+const spanner1 = new Spanner({
+  projectId: 'span-cloud-testing',
+  apiEndpoint: 'staging-wrenchworks.sandbox.googleapis.com',
+});
```

Comment:
I assume that this is temporary for the system tests

Reply:
Yes, will remove this.